networking/private-cloud-function-from-onprem/main.tf

```diff
@@ -49,69 +49,60 @@ module "project-gcp" {
 ###############################################################################
 
 module "vpc-onprem" {
-  source     = "github.com/terraform-google-modules/cloud-foundation-fabric//modules/net-vpc?ref=v5.0.0"
+  source     = "../../modules/net-vpc"
   project_id = module.project-onprem.project_id
-  name       = "onprem-net"
+  name       = "onprem"
   subnets = [
     {
-      ip_cidr_range      = var.onprem_ip_range
-      name               = "onprem-sub"
+      ip_cidr_range      = var.ip_ranges.onprem
+      name               = "onprem-subnet"
```

Comment:
`-subnet` is redundant in a subnet name, but I would use a prefix to make sure th ename does not clash with pre-existing subnets (and also use the prefix for all other resource names).